File: Code/cpp/cube.cpp
#include<iostream>
#include<cmath>
using namespace std;
int main()
{
    float edge;

    cout << "Enter Edge: ";
    cin >> edge;

    cout << "Volume: " << pow(edge, 3);
    cout << "\nSurface Area: " << 6 * pow(edge, 2);
    return 0;
}

File: Code/java/cube.java
import java.util.Scanner;

class Main
{
    public static void main(String[] args)
    {
        float edge;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Edge: ");
        edge = scanner.nextFloat();

        System.out.println("Volume: " + Math.pow(edge, 3));
        System.out.println("Surface Area: " + 6 * Math.pow(edge, 2));

        scanner.close();
    }
}